alerts/cluster-monitoring-operator/TargetDown.md

```diff
@@ -55,13 +55,42 @@ We are ready to diagnose the root cause by inspecting each component on the scra
 workflow.
 
 ### Potential Issues and Resolutions
+
+#### Unknown Content Type
+
+Beginning with OpenShift Container Platform version 4.19, Prometheus enforces stricter
+rules on the `Content-Type` header when scraping.
+
+If the scraped target does not have a `Content-Type` header or if the header is
+unparsable or unrecognised and no [`fallback_scrape_protocol`] is set for the
+target, the scrape fails, and the `TargetDown` alert is fired.
```

Comment:
```suggestion
target, the scrape fails and the `TargetDown` alert is fired.
```

Reply:
That was suggested by https://github.com/openshift/runbooks/pull/232#discussion_r1954421745